cmd/cli/app/quickstart/quickstart.go

```diff
@@ -143,7 +143,10 @@ var cmd = &cobra.Command{
 			return nil
 		}
 
-		// Step 1 - Confirm enrolling
+		// Step 1 - Confirm enrolling (new context, so we don't time out)
+		ctx, cancel := cli.GetAppContext(cmd.Context(), viper.GetViper())
```

Comment:
Just a question. Does this actually change the context deadline?

Reply:
yes and no. It doesn't actually change the timeout (the new patch I just pushed does), but what it does is instantiate a new context per step so that the old doesn't time out.

Reply:
I see. Since the output from the command requires some reading, would increasing the deadline slightly be a bad idea?

Reply:
that's what the new version of the patch I pushed does (doubles the timeout from 10 to 20 seconds for each step)